firmware/powerboard2/code/app/ADC.c

```diff
@@ -0,0 +1,149 @@
+/*
+ * ADC.c
+ */
+
+#include "ADC.h"
+
+/* INCLUDES */
+#include "stm32f4xx.h"
+#include <stdbool.h>
+
+/* DEFINES */
+
+/* TYPEDEFS */
+typedef struct
+{
+    uint16_t rawData[ADC_CHANNEL_COUNT];
+} ADC_data_S;
+
+/* PRIVATE DATA DECLARAION */
+static ADC_data_S ADC_data;
+
+static const uint32_t ADC_softwareChannelToADCChannelMapping[ADC_CHANNEL_COUNT] =
+{
+    [ADC_CHANNEL_ADC_BAT0_1]          = ADC_Channel_15,
+    [ADC_CHANNEL_ADC_BAT0_2]          = ADC_Channel_14,
+    [ADC_CHANNEL_ADC_BAT0_3]          = ADC_Channel_7,
+    [ADC_CHANNEL_ADC_BAT0_4]          = ADC_Channel_6,
+    [ADC_CHANNEL_ADC_BAT0_5]          = ADC_Channel_5,
+    [ADC_CHANNEL_ADC_BAT0_6]          = ADC_Channel_4,
+	[ADC_CHANNEL_ADC_BAT1_1]          = ADC_Channel_3,
```

Comment:
Format the lines. Looks cleaner that way

Reply:
The c file is fine. It's Github being stupid.